Mathlib/Order/Basic.lean

```diff
@@ -1503,7 +1502,7 @@ instance Prop.partialOrder : PartialOrder Prop where
 
 end «Prop»
 
-variable {s : β → β → Prop} {t : γ → γ → Prop}
+variable {γ : Type w} {s : β → β → Prop} {t : γ → γ → Prop}
```

Comment:
Why not keep `γ` at the top?

Reply:
It is used here for the first time. Why to clutter the infoview for the intermediate 1500 lines?

Reply:
Type variables are usually put at the top

Reply:
OK

Reply:
Actually, the entire line 1506 was never used, lol.

Reply:
Welp, that solves it